block-storage/src/main/scala/coop/rchain/blockstorage/BlockStoreSyntax.scala

```diff
@@ -31,19 +31,9 @@ final class BlockStoreOps[F[_]: Sync](
 
   /**
     * Get block, "unsafe" because method expects block already in the block store.
-    *
-    * Unfortunately there is no way to get stack trace when error is thrown in async execution.
-    * Monix does not have support and cats.effect support is in creation.
-    * https://github.com/typelevel/cats-effect/pull/854
-    * So extra source parameters are a desperate measure to indicate who is the caller.
     */
-  def getUnsafe(hash: BlockHash)(
-      implicit line: sourcecode.Line,
-      file: sourcecode.File,
-      enclosing: sourcecode.Enclosing
-  ): F[BlockMessage] = {
-    def source = s"${file.value}:${line.value} ${enclosing.value}"
-    def errMsg = s"BlockStore is missing hash ${PrettyPrinter.buildString(hash)}\n  $source"
```

Comment:
Why it is removed? It is not used or not important information?

Reply:
We had this to get source trace on errors because before monix v3 this was not supported so async code was impossible to trace.
https://github.com/monix/monix.io/blob/main/_docs/3x/eval/stacktraces.md